src/termData.js

```diff
@@ -10,7 +10,7 @@ function Term(shortName, longName, startDate) {
 // Quarterly Academic Calendar: https://www.reg.uci.edu/calendars/quarterly/2022-2023/quarterly22-23.html
 // Note: months are 0-indexed
 const termData = [
-    new Term('2023 Winter', '2023 Winter Quarter', [2023, 1, 4]),
+    new Term('2023 Winter', '2023 Winter Quarter', [2023, 0, 4]),
```

Comment:
```suggestion
    new Term('2023 Winter', '2023 Winter Quarter', [2023, 0, 9]),
```

Also, I think classes start on Monday the 9th, not Wednesday the 4th

Reply:
Yeah MB I used the "quarter begins" date instead of the "instruction begins" date.